src/IECore/DateTimeData.cpp

```diff
@@ -115,4 +115,11 @@ void SimpleDataHolder<boost::posix_time::ptime>::hash( MurmurHash &h ) const
 
 template class TypedData< boost::posix_time::ptime >;
 
+void murmurHashAppend( IECore::MurmurHash &h, const boost::posix_time::ptime &time )
+{
+	// Very sloppy, but appears to be complete even for nanosecond resolution ptimes
+	h.append( boost::posix_time::to_iso_string( time ) );
```

Comment:
Now we have this, we should remove the `SimpleDataHolder<boost::posix_time::ptime>::hash` specialisation from `DateTimeData.cpp`.

Reply:
Fixed squashed into 467a02b